src/main/scala/org/broadinstitute/clio/service/AuthorizationService.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.clio.service
+
+import akka.http.scaladsl.server.RequestContext
+import org.broadinstitute.clio.model._
+
+
+case class AuthorizationInfo(token: String, expiration: String, email: String, id: String)
+
+class AuthorizationService() {
+
+  def authorize(context: RequestContext): Boolean = {
+    val pairs = for (header <- context.request.headers) yield
+      header match {
+        case OidcAccessToken(token)      => 0 -> token
```

Comment:
The numbers just represent parameters to AuthorizationInfo.
I hope there is a solution better than yours (and mine).